### ISSUE TITLE
Add new assets for Spacewalk on Pendulum

### ISSUE THREAD

**Issue by annatekl** [pendulum-chain/portal#415]

**Comment by annatekl** [comment]
@prayagd do you need the icons ?

**Comment by vadaynujra** [comment]
@annatekl I see that this ticket is for the portal. Don't we need a ticket to actually enable the assets on Spacewalk, before enabling it on the UI? Could you create that ticket? You can use [this](https://github.com/pendulum-chain/tasks/issues/87) ticket for reference.

**Comment by vadaynujra** [comment]
Let also add the EURC (Circle) to the list?

**Comment by prayagd** [comment]
Putting this on hold until the onchain token is not added

**Comment by ebma** [comment]
This ticket is unblocked now since the asset was added to the on-chain data. 
The Circle EURC.s is actually already shown in the dashboard but it's using the same token as the Mykobo one so we need to fix that. I propose we make this ticket about changing the icon and that's it.

**Comment by annatekl** [comment]
covered by this [ticket](https://app.zenhub.com/workspaces/pendulum-6408a0365cda59c57d39bee5/issues/gh/pendulum-chain/portal/495). Shall we close this one @ebma

**Comment by ebma** [comment]
Yes true, let's close this ticket then.
